lib/capability/capability_test.go

```diff
@@ -0,0 +1,275 @@
+package capability
+
+import (
+	"testing"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+func TestSetCapabilities(t *testing.T) {
+	tests := []struct {
+		name            string
+		config          *configv1.ClusterVersion
+		wantKnownKeys   []string
+		wantEnabledKeys []string
+	}{
+		{name: "capabilities nil",
+			config:          &configv1.ClusterVersion{},
+			wantKnownKeys:   []string{string(configv1.ClusterVersionCapabilityOpenShiftSamples)},
+			wantEnabledKeys: []string{string(configv1.ClusterVersionCapabilityOpenShiftSamples)},
```

Comment:
This will break as folks continue to add caps, like openshift/api#1125.  Speaking of which, we will need to bump our vendored API version again, although I don't mind if we delay that until we get closer to feature freeze.  But anyway, I expect we want to use `configv1.ClusterVersionCapabilitySets[configv1.ClusterVersionCapabilitySetCurrent]` as the default value for enabled keys.  And probably for the time being assume that will be the full set of known keys too?  Otherwise this feels a lot like it's re-implementing `setKnownCapabilities`.